addons/website_event/static/src/js/website_event.js

```diff
@@ -52,7 +52,7 @@ var EventRegistrationForm = Widget.extend({
             return new Promise(function () {});
         } else {
             $button.attr('disabled', true);
-            return ajax.jsonRpc($form.attr('action'), 'call', post).then(function (modal) {
+            return ajax.jsonRpc($form.data('action'), 'call', post).then(function (modal) {
```

Comment:
What would happen if website_event module is not updated? Wouldn't this fail since there is no `data-action`? Maybe this should be `$form.data('action') || $form.attr('action') ` before master

Reply:
@nle-odoo If these two files are for whatever reason not updated in sync, this might be necessary. Are you aware of any such cases?

Reply:
JS and XML files are updated in sync in the filesystem.

But for the database:

- JS files are updated without update of the modules (the hash of the asset bundle should change when a JS file is changed and so the bundle will be updated and should be automatically up to date)

- XML files used by python (which is the case here) are only updated if you update the module

so as far as I can tell, in this case the JS will be updated and the XML will not for the majority of instance (besides those that update or install website_event after the fix is available).